terraform/modules/vpc/main.tf

```diff
@@ -1,13 +1,14 @@
 module "vpc" {
   source = "terraform-aws-modules/vpc/aws"
+  version = "10.0.0"
 
   name = "${var.project}-${var.environment}-vpc"
-  cidr = "10.10.0.0/16"
-
-  azs              = ["${var.region}a", "${var.region}b", "${var.region}c"] # Most regions have 3+ azs
-  private_subnets  = ["10.10.32.0/19", "10.10.64.0/19", "10.10.96.0/19"]
-  public_subnets   = ["10.10.1.0/24",  "10.10.2.0/24",  "10.10.3.0/24"]
-  database_subnets = ["10.10.10.0/24", "10.10.11.0/24", "10.10.12.0/24"]
+  cidr = "10.11.0.0/16"
```

Comment:
OOC why did we have to change this octet?

Reply:
Oops, I missed a couple of my TODO comments to remove some stuff I changed for testing. Will fix, thanks.